crmsh/utils.py

```diff
@@ -2018,6 +2013,51 @@ def _cloud_metadata_request(uri, headers={}):
         return None
 
 
+def detect_aws():
+    """
+    Detect if in AWS
+    """
+    system_version = get_stdout_or_raise_error("dmidecode -s system-version")
```

Comment:
Most probably AWS as well has some internal metadata REST endpoint.
If you want to have a look

Reply:
Maybe let's make this improvement later, in other PR?
I don't have aws environment in hand now

Anyway, thanks for suggestions!

Reply:
Yes, of course!
I just thought that It was a relevant comment hehe